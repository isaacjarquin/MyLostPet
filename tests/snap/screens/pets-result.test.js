import 'react-native'
import React from 'react'
import PetsResult from '../../../screens/pets-result'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const pet = {name: 'name'}
  const navigation = {
    state: {
      params: {
        pets: [pet]
      }
    }
  }

  const tree = renderer.create(
    <PetsResult navigation={navigation} />
    ).toJSON()
  expect(tree).toMatchSnapshot()
})
