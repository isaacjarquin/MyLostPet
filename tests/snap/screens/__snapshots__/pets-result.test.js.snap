// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "flex-end",
    }
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#bbb",
          "borderTopWidth": 1,
          "marginTop": 20,
        },
        undefined,
      ]
    }
  >
    <RCTScrollView
      dataSource={
        ListViewDataSource {
          "items": 1,
        }
      }
      renderRow={[Function]}
      renderScrollComponent={[Function]}
    >
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderBottomColor": "#bbb",
                  "borderBottomWidth": 1,
                  "paddingBottom": 10,
                  "paddingRight": 10,
                  "paddingTop": 10,
                },
                Object {
                  "paddingLeft": 10,
                },
              ],
            ]
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "marginLeft": 10,
                },
                undefined,
              ]
            }
          >
            <View
              style={
                Object {
                  "height": 34,
                  "width": 34,
                }
              }
            >
              <View
                activeOpacity={undefined}
                onLongPress={undefined}
                onPress={undefined}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "height": 34,
                      "paddingBottom": 10,
                      "paddingRight": 10,
                      "paddingTop": 10,
                      "width": 34,
                    },
                    Object {
                      "borderRadius": 17,
                    },
                    undefined,
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "alignSelf": "stretch",
                        "backgroundColor": "rgba(0,0,0,0.2)",
                        "bottom": 0,
                        "flex": 1,
                        "justifyContent": "center",
                        "left": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      },
                      Object {
                        "borderRadius": 17,
                      },
                      undefined,
                    ]
                  }
                >
                  <Image
                    source={
                      Object {
                        "uri": undefined,
                      }
                    }
                    style={
                      Array [
                        Object {
                          "height": 34,
                          "width": 34,
                        },
                        Object {
                          "borderRadius": 17,
                        },
                        Object {
                          "borderRadius": 27,
                          "height": 55,
                          "marginLeft": 20,
                          "marginRight": 20,
                          "marginTop": 20,
                          "width": 55,
                        },
                      ]
                    }
                  />
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "flex": 1,
                  "justifyContent": "center",
                }
              }
            >
              <View
                style={
                  Object {
                    "marginLeft": 10,
                  }
                }
              >
                <View />
              </View>
              <View
                style={
                  Object {
                    "marginLeft": 10,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  numberOfLines={2}
                  style={
                    Array [
                      Object {},
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      Array [
                        Object {
                          "color": "#86939e",
                          "fontSize": 15,
                          "fontWeight": "600",
                          "marginTop": 1,
                        },
                        Object {
                          "marginLeft": 10,
                        },
                        undefined,
                        undefined,
                      ],
                    ]
                  }
                >
                  Encontrado en undefined, el undefined. undefined
                </Text>
              </View>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              collapsable={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "flex-end",
                  "flex": 0.15,
                  "justifyContent": "center",
                  "opacity": 1,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                onPress={undefined}
                style={
                  Array [
                    false,
                    false,
                    false,
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "justifyContent": "center",
                    },
                    undefined,
                  ]
                }
                underlayColor="white"
              >
                <Text
                  accessible={true}
                  allowFontScaling={false}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {
                        "color": "#bdc6cf",
                        "fontSize": 28,
                      },
                      Array [
                        Object {
                          "backgroundColor": "transparent",
                        },
                        undefined,
                      ],
                      Object {
                        "fontFamily": "Material Icons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                    ]
                  }
                >
                  î—Œ
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
  <View
    style={
      Object {
        "marginTop": 10,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#393e42",
            "borderBottomColor": "#000",
            "borderBottomWidth": 1,
            "borderTopColor": "#000",
            "borderTopWidth": 1,
          },
          false,
          undefined,
        ]
      }
    >
      <TextInput
        placeholder="Ciudad/Municipio..."
        placeholderTextColor="#86939e"
        selectionColor="#86939e"
        style={
          Array [
            Object {
              "backgroundColor": "#303337",
              "borderRadius": 3,
              "color": "#86939e",
              "fontSize": 17.5,
              "height": 30,
              "margin": 8,
              "overflow": "hidden",
              "paddingLeft": 26,
              "paddingRight": 19,
            },
            false,
            false,
            Object {
              "borderRadius": 15,
            },
            undefined,
            undefined,
            false,
          ]
        }
        underlineColorAndroid="transparent"
      />
      <Text
        accessible={true}
        allowFontScaling={false}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#86939e",
              "fontSize": 16,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
                "position": "absolute",
                "top": 15.5,
              },
              Object {
                "left": 16,
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î¢¶
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#393e42",
            "borderBottomColor": "#000",
            "borderBottomWidth": 1,
            "borderTopColor": "#000",
            "borderTopWidth": 1,
          },
          false,
          undefined,
        ]
      }
    >
      <TextInput
        placeholder="Raza..."
        placeholderTextColor="#86939e"
        selectionColor="#86939e"
        style={
          Array [
            Object {
              "backgroundColor": "#303337",
              "borderRadius": 3,
              "color": "#86939e",
              "fontSize": 17.5,
              "height": 30,
              "margin": 8,
              "overflow": "hidden",
              "paddingLeft": 26,
              "paddingRight": 19,
            },
            false,
            false,
            Object {
              "borderRadius": 15,
            },
            undefined,
            undefined,
            false,
          ]
        }
        underlineColorAndroid="transparent"
      />
      <Text
        accessible={true}
        allowFontScaling={false}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#86939e",
              "fontSize": 16,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
                "position": "absolute",
                "top": 15.5,
              },
              Object {
                "left": 16,
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î¢¶
      </Text>
    </View>
  </View>
</View>
`;
